istioctl/cmd/root.go

```diff
@@ -92,6 +92,7 @@ debug and diagnose their Istio mesh.
 	rootCmd.AddCommand(register())
 	rootCmd.AddCommand(deregisterCmd)
 	rootCmd.AddCommand(injectCommand())
+	rootCmd.AddCommand(uninjectCommand())
```

Comment:
Should we put this function under 'experimental' until we have some feedback on using it?

Reply:
Great comment. Thanks.